docs/integrations/chalice.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chalice
+---
+
+# Chalice
+
+Strawberry comes with a chalice integration. It provides a view that you can
+use to serve your GraphQL schema:
+
+Run chalice command line to create a new project
+```shell
+$ chalice new-project badger-project
+$ cd badger-project
+```
+
+replace the contents of app.py with the following:
+
+```python
+from chalice import Chalice
+from chalice.app import Request
+
+import strawberry
+from strawberry.chalice.views import GraphQLView
+
+app = Chalice(app_name="BadgerProject")
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def greetings(self) -> str:
+        return "hello from the illustrious stack badger"
+
+
+@strawberry.type
+class Mutation:
+    @strawberry.mutation
+    def echo(self, string_to_echo: str) -> str:
+        return string_to_echo
+
+
+schema = strawberry.Schema(query=Query, mutation=Mutation)
+view = GraphQLView(schema=schema, render_graphiql=True)
+
+
+@app.route("/graphql", methods=["GET", "POST"], content_types=["application/json"])
+def handle_graphql():
```

Comment:
Type annotate the return type here (might just be `object`?)

Reply:
Done